docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -89,10 +86,10 @@ NOTE: Only some basic redirect is supported namely a GET. Only the URL and query
 
 OpenID Client can be configured with both `@OpenIdAuthenticationDefinition` annotation attributes and Microprofile Config properties.
 
-CAUTION: When you provide configuration using MicroProfile Config properties and you use a reference to a variable provided by Payara using the `${}` notation inside the values, you need to escape the `$` sign with `\\`. Otherwise they will be treated as references to other MicroProfile Config properties, which may result in an error if such a MicroProfie property isn't found. This especially applies to the `baseURL` variable, which is only accepted in the value of the `redirectURI` option. For example, to specify `redirectURI` using a MicroProfile Config property which is equal to `${baseURL}/Callback` used in the annotation, you need to use the following MicroProfile Config key=value definition: `payara.security.openid.redirectURI=\\${baseURL}/Callback`.
+CAUTION: If you are using MicroProfile Config properties and the value contains a placeholder that should not be resolved by MicroProfile Config
+itself but by the Server (like `payara.security.openid.redirectURI=${baseURL}/Callback`) make sure the value is properly escaped. It will otherwise result in `NoSuchElementException`. Here's an example how such a value should be escaped: `payara.security.openid.redirectURI=\\${baseURL}/Callback`
 
-The annotation and MicroProfile properties has several configuration options.
-They are detailed below.
+The annotation and MicroProfile properties have several configuration options. They are detailed below.
 
 .Configuration Options
```

Comment:
```suggestion
.Configuration Options For `@OpenIdAuthenticationDefinition`
```
I suggest that we make it clear that this table is for the main annotation and we add another table under this table with options for the `@OpenIdProviderMetadata` annotation too. Currently we have at least 2 options to document, which are scattered through this documentation: `disableScopeValidation` (mentioned in a separate section) and `endSessionEndpoint` mentioned in the description of the `notifyProvider` option.

Reply:
already fixed